website/templates/emails/pending_retraction_admin.txt.mako

```diff
@@ -0,0 +1,15 @@
+Hello ${user.fullname},
+
+
+I just wanted to let you know, ${initiated_by} has initiated a retraction for the following registration: ${registration_link}
```

Comment:
Oh, perhaps that is for a registration.  Here:

I just wanted to let you know, ${initiated_by} has initiated a retraction for the following registration: ${registration_link}.
+
+If approved, the existing registration will be retracted leaving only [same text as elsewhere for explaining what retraction does]. 

Only project administrators can initiate or cancel a retraction.  Each project administrator will have received this email.  If all approve or do not respond within ${approval_time_span} hours from midnight tonight (ET), the retraction will proceed.  If one disapproves within ${approval_time_span} hours from midnight tonight (ET), the retraction will be canceled. 
+
+To approve this action click the following link: ${approval_link}.
+
+To disapprove this action, click the following link: ${disapproval_link}.
+
+Note: Clicking the disapproval link will immediately cancel the pending retraction. If you neither approve nor disapprove the embargo within ${approval_time_span} hours from
+midnight tonight (EDT) the retraction will proceed.
+
+Sincerely yours,
+
+The OSF Robot